package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "bluedot-react-native",
   "title": "React Native Bluedot Point SDK",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
The current version in [NPM](https://www.npmjs.com/package/bluedot-react-native) is `2.0.1`, can you bump it to `2.0.2`?

Reply:
version updated

Reply:
Thanks!